Extensions/ParticleSystem/particleemitterobject.ts

```diff
@@ -158,8 +160,13 @@ namespace gdjs {
       this.particleRotationMaxSpeed = particleObjectData.particleAngle2;
       this.maxParticlesCount = particleObjectData.maxParticleNb;
       this.additiveRendering = particleObjectData.additive;
+      this.jumpForwardInTimeOnCreation =
+        particleObjectData.jumpForwardInTimeOnCreation;
       this._textureDirty = this.texture !== '';
 
+      // Jump emitter forward in time
+      this._renderer.update(this.jumpForwardInTimeOnCreation);
```

Comment:
Try to add a console.log here to see if `this.jumpForwardInTimeOnCreation` has a proper value?

My hypothesis is that it's maybe too early to ask the renderer to update?